_layouts/base.html

```diff
@@ -6,6 +6,16 @@
   <title>{{ title or "Architecture Of Konflux" }}</title>
   <meta name="description" content="{{ description or 'Technical documents about the Managed Developer Platform' }}">
 
+  <!-- Google Analytics tracking code -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-381LYFWTP4"></script>
```

Comment:
Do we need to ask for consent from the user before running this?

Reply:
I think we do, and probably fill-in one of those three-letter forms about privacy protections.